devcycle.ts

```diff
@@ -0,0 +1,14 @@
+import { DVCClient, initialize } from "@devcycle/nodejs-server-sdk";
+
+const sdk = process.env.DEVCYCLE_TOKEN as string;
+
+const globalForDevCycle = global as unknown as { devCycle: DVCClient };
+
+const getDVCClient = async () => {
+  if (typeof globalForDevCycle.devCycle === "undefined") {
+    globalForDevCycle.devCycle = await initialize(sdk).onClientInitialized();
+  }
+  return globalForDevCycle.devCycle;
+};
+
+export default getDVCClient;
```

Comment:
Does this work?

```suggestion
import { DVCClient, initialize } from "@devcycle/nodejs-server-sdk";

const sdkKey = process.env.DEVCYCLE_CLIENT_SDK_KEY as string;

let dvcClient: DVCClient

export default const getDVCClient = async () => {
  if (!dvcClient) {
    dvcClient.devCycle = await initialize(sdkKey).onClientInitialized()
  }
  return dvcClient
};
```

Reply:
Why is the Client SDK key enough here? I'm using the NodeJS Server SDK to be able to do local bucketing for all users. This shouldn't be possible with the Client SDK key.

For the client I want to use the React SDK and there I would need the Client CDK Key in order to fetch data only for one particular user.

Do I have a misunderstanding here?